CMakeLists.txt

```diff
@@ -97,7 +97,6 @@ list(APPEND CMAKE_PREFIX_PATH  # Add ROCM_PATH to CMake search paths (for findin
             ${ROCM_PATH}
             ${ROCM_PATH}/hip
             ${ROCM_PATH}/llvm
-            ${ROCM_PATH}/hcc)
```

Comment:
Hey does this not mean that you now have an open parenthesis?

Reply:
yep - agreed. thanks for catching that.